grt-eval.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+################################################################################
+#
+# This script runs Defects4j computation in parallel. It handles setup (excluding
+# Defects4j-specific setup, which you should do from the README.md of this repo).
+# The command should be run from the root directory of `defects4j-grt`.
+# 
+# Use ./grt-eval.sh --ignore-warning to bypass the user check
+# This can be useful when making this a background process.
+# 
+# If there are java versions or perl libraries missing, see grt-eval-setup.sh
+#
+################################################################################
+
+# Function to prompt the user for confirmation
+confirm_proceed() {
+    while true; do
+        read -p "Type 'y' to continue: " response
+        case $response in
+            [Yy]* )
+                echo "Proceeding with the action."
+                return 0  # Success, proceed with the action
+                ;;
+            [Nn]* )
+                echo "Action aborted."
+                return 1  # Exit or abort, action canceled
+                ;;
+            * )
+                echo "Invalid input. Please type 'y' to proceed or 'n' to cancel."
+                ;;
+        esac
+    done
+}
+
+# Functions to switch jdk version
+usejdk8() {
+    export JAVA_HOME=~/java/jdk8u292-b10
+    export PATH=$JAVA_HOME/bin:$PATH
+}
+usejdk11() {
+    export JAVA_HOME=~/java/jdk-11.0.9.1+1
+    export PATH=$JAVA_HOME/bin:$PATH
+}
+
+# Warn about files removed
+echo "Warning: This script will REMOVE all defects4j-grt/framework/test/[test_d4j_*|*.log] directories and files!"
+echo "Are you sure you want to proceed? (y/n)"
+if [[ "$1" == "--ignore-warning" ]] || confirm_proceed; then
+  echo "Running script..."
+else
+  exit 1
+fi
+
+# Setup java versions
+# TODO how to set this up without sudo perms?
+
+# Setup Defects4j
+echo "START: Setting up defects4j-grt"
+
+cpanm --installdeps .
+./init.sh
+
+cd ..
+WORK_DIR=$(pwd)
+export D4J_HOME=$WORK_DIR/"defects4j-grt"
+export PATH=$PATH:$D4J_HOME/"framework/bin"
+export randoop=$WORK_DIR/"randoop-grt"
```

Comment:
There are some assumptions about the directory structure when the script is called.  Please document them.